src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,215 @@
+@import '../../variables' ;
+
+.footer {
+    flex-shrink: 0;
+    display: block;
+}
+
+.pt-5 {
+    padding-top: 3rem !important;
+}  
+
+.col-md-4 {
+    flex: 0 0 auto;
+    width: 33.33333333%;
+}
+
+.widget-title {
+    margin-bottom: 1.125rem;
+    font-size: 1.0625rem;
+    font-weight: 600;
+}
+
+.widget-list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    &[data-simplebar] {
+        overflow-x: hidden;
+    }
+}
+
+.widget-list-item {
+    margin-bottom: 0.5rem;
+    list-style-type: none;
+    a:hover{
+        color: whitesmoke;
+        font-weight: bold;
+    }
+}
+
+.widget-list-link {
+    display: block;
+    color: rgba(255,255,255,.65);
+    font-size: .875rem;
+    font-weight: normal;
+    text-decoration: none;
+}
+
+.widget{
+    color: #f7f9fd;
+    margin: 0px 0px 24px;
```

Comment:
Try to avoid using of px; use rem instead

Reply:
@akashsinha13, modified to rem instead of px.